architecture/job.go

```diff
@@ -119,10 +112,10 @@ func (j *Job) Key() int64 {
 	case DELAYED:
 		// time remaining from Delay till it gets ready becomes priority
 		// log.Println(j.Delay * NANO, time.Now().UnixNano(), j.StartedDelayAt)
-		return j.Delay * NANO - (time.Now().UnixNano() - j.StartedDelayAt)
+		return j.Delay*NANO - (time.Now().UnixNano() - j.StartedDelayAt)
```

Comment:
`go fmt` eliminated it.  If you do want the spacing back, let me know and I'll do it.